tests/e2e/config/jest.setup.js

```diff
@@ -11,8 +11,8 @@ import {
 	visitAdminPage,
 } from '@wordpress/e2e-test-utils';
 
-// Set the default test timeout to 30s.
-let jestTimeoutInMilliSeconds = 30000;
+// Set the default test timeout to 40s.
+let jestTimeoutInMilliSeconds = 40000;
```

Comment:
Do we still need this timeout increase now? It shouldn't be needed?

Reply:
Nope, Ill revert this.